sentry-servlet-jakarta/src/main/java/io/sentry/servlet/jakarta/SentryServletRequestListener.java

```diff
@@ -21,6 +23,8 @@
 @Open
 public class SentryServletRequestListener implements ServletRequestListener {
 
+  public static final String SENTRY_LIFECYCLE_TOKEN_KEY = "sentry-lifecycle";
```

Comment:
`nit`: How about making it more clear this key takes care of the scope lifecycle?
```suggestion
  public static final String SENTRY_SCOPE_LIFECYCLE_TOKEN_KEY = "sentry-scope-lifecycle";
```